esphome/components/esp32/gpio_esp32_c3.py

```diff
@@ -0,0 +1,61 @@
+import logging
+
+from esphome.const import (
+    CONF_INPUT,
+    CONF_MODE,
+    CONF_NUMBER,
+    CONF_PULLDOWN,
+    CONF_PULLUP,
+)
+import esphome.config_validation as cv
+
+_ESP_32_C3_SPI_PSRAM_PINS = {
```

Comment:
```suggestion
_ESP32C3_SPI_PSRAM_PINS = {
```

For consistency, in the rest of the codebase we usually write it together and without spaces, and IMO that's also easier to read